strapi/strapi
# fix: translation of profile language setting

### What does it do?

![SCR-20221223-niy](https://user-images.githubusercontent.com/20895743/209305828-8805560c-244d-4308-96f5-f9e988b9a255.png)

After PR #12788, the translation of profile language setting is broken since the i18n key is changed from `Settings.profile.form.section.experience.documentation` to `Settings.profile.form.section.experience.here`.

But first, one line is forgot to change
https://github.com/strapi/strapi/blob/05be5849a7e12707af02e4351bc94edca35d89f4/packages/core/admin/admin/src/pages/ProfilePage/index.js#L461

Secondly, it breaks many non-EN translations if no one updated that i18n key.

This PR aimed to fix both issues.

### How to test it?

Go to the profile language setting page, change to different languages, see if the highlighted hint change accordingly.

### Related issue(s)/PR(s)

#12788
